#!/usr/bin/env python
PACKAGE = "vision"

from dynamic_reconfigure.parameter_generator_catkin import *

gen = ParameterGenerator()

# Params for gate task
gen.add("loOrange", int_t, 0, "Low  bound for Orange", 0, 0, 255)
gen.add("hiOrange", int_t, 0, "High  bound for Orange", 29, 0, 255)

gen.add("minContourArea",int_t,0,"Minimum threshold area for gate contours",194,0,1000)


exit(gen.generate(PACKAGE, "gate_task", "gate"))
